README.md

```diff
@@ -31,23 +34,70 @@ To run in watch mode, add `-w`/`--watch` to the command:
 npx steiger ./src --watch
 ```
 
-# Configuration
+## Configuration
 
 Steiger is zero-config! If you don't want to disable certain rules, you can safely skip this section.
 
 Steiger is configurable via `cosmiconfig`. That means that you can create a `steiger.config.ts` or `steiger.config.js` file in the root of your project to configure the rules. Import `{ defineConfig } from "steiger"` to get autocompletion.
 
-```ts
-import { defineConfig } from 'steiger'
+The config file shape is highly inspired by ESLint's config file, so if you have configured ESLint before, you'll find it easy to configure Steiger.
+
+### Example
+
+Here are some rules on how configuration is processed:
 
-export default defineConfig({
-  rules: {
-    'public-api': 'off',
+- Config objects are processed from top to bottom, so if there are multiple config object that match the same file for the same rule, the last one will be applied.
+- GlobalIgnore objects `{ ignores: ['**/__mocks__/**'] }` are applied to all rules, they are processed first and permanently remove files from the linter's field of view, so you can't reassign severity to them in other later objects.
+- You can set options for a rule once. When set, options are applied for the entire file system that is covered by Steiger.
+
+Note that this line `...fsd.configs.recommended,` just takes the plugin and the recommended rules configuration (all enabled with "error" severity by default) and puts it into the config array.
```

Comment:
suggestion: let's not try to document every single aspect of behaviour. I think this makes instructions too big, which means that people will be less likely to read them.

For instance:
- "Config objects are processed from top to bottom" — I think this is something that they can discover by trying, and most people are probably not even going to encounter that
- "GlobalIgnore objects" — I think this goes a bit too deep into the internal terminology like "GlobalIgnore", "assign severity", "linter's field of view", I don't think that's super relevant. I suggest simplifying it
- "You can set options for a rule once." — at present, none of the rules even accept options, I think, so this is very much irrelevant :D
- About the `fsd.configs.recommended`, I don't think we need to tell people how it works exactly, all they need to know is that it will set up the linter the way we recommend, and they can tweak it further if they want.

I don't even think it should be a list, let's just throw a tip like this after that simple example:
```suggestion
> [!TIP]
> If you want Steiger to ignore certain files, add an object like this to the config array: 
> ```js
> defineConfig([
>   /* … */, 
>   { ignores: ['**/__mocks__/**'] }
> ])
> ```
```

Reply:
I agree, most of this section is either irrelevant or too complex